devlab/tests/test_resource_migration.py

```diff
@@ -150,12 +165,17 @@ def test_migrate_neutron_routers(self):
             src_routers, dst_routers, resource_name='routers')
 
     def test_migrate_vms_parameters(self):
-        src_vms = self.src_cloud.novaclient.servers.list(
+        src_vms_names = self.src_cloud.novaclient.servers.list(
             search_opts={'all_tenants': 1})
-        dst_vms = self.dst_cloud.novaclient.servers.list(
+        dst_vms_names = self.dst_cloud.novaclient.servers.list(
             search_opts={'all_tenants': 1})
+        src_vms = [x.__dict__ for x in src_vms_names]
+        dst_vms = [x.__dict__ for x in dst_vms_names]
+
+        filtering_data = self.migration_utils.filter_objects(src_vms, 'vm')
```

Comment:
Do not convert `src_vms_names` to `src_vms`, just push whatever nova returns here.